### ISSUE TITLE
Delete channel

### ISSUE THREAD

**Issue by Panda25cutie** [erfan4lx/Telegram-Channel-Report-Sender#1]
The owner of that channel is bullying me by creating fake screenshots

**Comment by erfan4lx** [comment]
Hey that's why we're selling this program, if you need I'm at your service.

**Comment by Panda25cutie** [comment]
Yes I need ure help as those channel owners r harassing me n my friends

**Comment by erfan4lx** [comment]
Sure, kindly contact us through Telegram or Email.
https://t.me/erfan4lx10
erfan4lx@gmail.com
